### PR TITLE
fix test imports

### PR DESCRIPTION
**Pull request by griest024**
also exports the filter errors

### CODE REVIEW

File: libs/category/src/filters/public_api.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export * from './to-request/public_api';
 export * from './type/range/public_api';
 export * from './type/equal/public_api';
 export * from './validators/public_api';
```
Comment (damienwebdev) [inline libs/category/src/filters/public_api.ts:10]: I don't think the errors should be exported here, they should only be from the root errors API.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/category/src/filters/public_api.ts (hunk 1 above)
Comment (griest024) [reply libs/category/src/filters/public_api.ts]: moved export

Comment (damienwebdev) [reply libs/category/src/filters/public_api.ts]: I misunderstood what was in the API of filter errors, my mistake. They can be exported from the filters public API.

Comment (griest024) [reply libs/category/src/filters/public_api.ts:10]: reverted

**Pull request merged**